client/css/style.css

```diff
@@ -1178,19 +1171,14 @@ kbd {
 
 #chat .toggle-button {
 	display: inline-block;
-	color: #666;
-	transition: color .2s, transform .2s;
+	transition: opacity .2s, transform .2s;
 }
 
-#chat .toggle-button.opened {
+#chat .toggle-button.opened, /* Thumbnail toggle */
+#chat .msg.condensed:not(.closed) .toggle-button { /* Expanded status message toggle */
 	transform: rotate(90deg);
 }
 
-#chat .toggle-button:hover {
-	/* transform and opacity together glitch, so need to use RGBA transition */
-	color: rgba(102, 102, 102, .8); /* #666 x .8 opacity */
-}
-
```

Comment:
I removed this because it became impossible to deal with knowing that opacity was applied directly on the button for thumbnail toggles, but on the parent element for condensed toggle. More importantly, the glitch was still present with this hack, and reverting this mostly went back to an un-glitchy state :man_shrugging:.